smart_home/devices/adjustable_device.py

```diff
@@ -2,29 +2,42 @@
 
 from smart_home.devices.switchable_device import SwitchableDevice
 from smart_home.interfaces.adjustable_device_interface import AdjustableDeviceInterface
+from smart_home.logging.logger import setup_logger
 
 
 class AdjustableDevice(SwitchableDevice, AdjustableDeviceInterface, ABC):
+    logger = setup_logger('AdjustableDevice')
 
     def __init__(self, name: str, initial_level: float = 1.0, min_level: float = 0.0, max_level: float = 1.0):
         super().__init__(name)
         self.min_level = min_level
         self.max_level = max_level
         self.value = initial_level
+        self.logger.info(f'AdjustableDevice {name} created with initial level {initial_level}, min level {min_level} '
```

Comment:
Um es einfach zu halten könnten wir doch die loggers nur in die Abstrakten klassen hinzufügen und es bei den spezifischen Implementierungen weg lassen? Bzw. anderes herum. Bei beiden Fällen steht dann nicht das gleiche Device zwei mal in den logs.

Reply:
Ja, ich dachte mir aber, dass dadurch in den Logs die Abstraktionsebene etwas klarer wird und wie die Operation durch den Stack wandert

Reply:
Hab's geändert. Die Abstrake Klasse instanziiert den Logger aber die anderen Logging-Nachrichten der jeweiligen unterklassen bleiben unverändert.